lib/tasks/pulmap.rake

```diff
@@ -89,6 +89,21 @@ namespace :pulmap do
     end
   end
 
+  namespace :delete_record do
+    desc "Delete a single record"
+    task :index, :id do |_t, args|
+      raise ArgumentError, "An id argument is required" unless args.id
+
+      # Set up connection and args
+      solr = Blacklight.default_index.connection
+      id = args[:id]
```

Comment:
suggestion: I would move this above line 95 and then use `unless id` instead of `unless args.id`